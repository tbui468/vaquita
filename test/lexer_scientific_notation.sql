if exists drop database sol;
create database sol;
open sol;

create table planets (
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        ("Venus", 4.87e2, 0, 92.0, false),
        ("Earth", -5.97e-2, 1, 1.0, false),
        ("Mars", 0.23e-2, 2, 0.01, false);

select * from planets;
close sol;
drop database sol;
exit;
