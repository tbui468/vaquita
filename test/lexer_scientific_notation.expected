created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
1, Venus, 487.000000, 0, 92.000000, false
2, Earth, -0.059700, 1, 1.000000, false
3, Mars, 0.002300, 2, 0.010000, false
closed database sol
dropped database sol
